R/flatten.R

```diff
@@ -1,58 +1,53 @@
-#' Flatten part of a `dm` into a wide table
+#' Flatten a part of a `dm` into a wide table
 #'
-#' Gather all information of interest in one place in a wide table (on a database-[`dm`] a
-#' temporary table will be created).
-#' If referential integrity is given among the tables of the data model, the resulting
-#' table of this function will contain as many rows as the table `start` does (exceptions are
-#' `join = anti_join` (result is empty table with same columns as `start`) and `join = right_join`
-#' (number of rows equal to or larger than those of `start`)).
+#' Gather all information of interest in one place in a wide table - a temporary table that will be created on a database-[`dm`].
```

Comment:
This was incorrect in the original.

All operations return "lazy tables" for database dm-s. Only `compute()` or `copy_to()` actually materialize tables, by default as temporary tables. This design principle is consistent with similar APIs but can be mentioned where it makes sense; this particular operation here isn't specific enough to warrant this.

```suggestion
#' Gather all information of interest in one place in a wide table.
```